src/js/tracks/text-track.js

```diff
@@ -19,7 +19,25 @@ import XHR from 'xhr';
  * @param {String} srcContent webVTT file contents
  * @param {Track} track track to addcues to
  */
-const parseCues = function(srcContent, track) {
+const parseCues = function(srcContent, track, retryNum) {
+  // Make sure that vttjs has loaded, otherwise, wait till it finished loading
+  // NOTE: this is only used for the alt/video.novtt.js build
+  if (typeof window.WebVTT !== 'function') {
+    let retry;
+
+    if (retryNum === 0) {
+      return log.error(`vttjs did not load, stopping trying to process ${track.src}`);
+    } else if (typeof retryNum !== 'number') {
+      retry = 3;
+    } else {
+      retry = retryNum - 1;
+    }
+
+    return window.setTimeout(function() {
+      parseCues(srcContent, track, retry);
+    }, 100);
+  }
```

Comment:
I could also have it do incremental back-off, 100 for first, 200 for second, 400 for 3rd, 800 for second. Total would be 1.5s.
Assuming we don't just do the onload.